content/2_Bronze/Intro_Sets.mdx

```diff
@@ -755,3 +633,223 @@ Some of these problems can be solved by sorting alone, though sets or maps could
 make their implementation easier.
 
 <Problems problems="standard" />
+
+## Check Your Understanding
+
+<LanguageSection>
+<CPPSection>
+
+<Quiz>
+  <Quiz.Question>
+    What is the time complexity of insertions, deletions, and searches in a sorted set of size $N$?
+    <Quiz.Answer>
+      $\mathcal{O}(1)$
+    </Quiz.Answer>
+    <Quiz.Answer>
+      $\mathcal{O}(N \log N)$
+    </Quiz.Answer>
+    <Quiz.Answer>
+      $\mathcal{O}(N)$
+    </Quiz.Answer>
+    <Quiz.Answer correct>
+      $\mathcal{O}(\log N)$
+      <Quiz.Explanation>
+		Correct.
+      </Quiz.Explanation>
+    </Quiz.Answer>
+  </Quiz.Question>
+  <Quiz.Question>
+    Which of the following is the correct output of this code?
+	```cpp
+	set<int> s;
+	s.insert(1);
+	s.insert(4);
+	s.insert(5);
+	s.insert(2);
+	cout << s.count(3) << endl;
+	```
+    <Quiz.Answer>
+      $1$
+      <Quiz.Explanation>
+
+      </Quiz.Explanation>
+    </Quiz.Answer>
+    <Quiz.Answer correct>
+      $0$
+      <Quiz.Explanation>
+        Correct.
```

Comment:
don't delete the explanation, just do `Correct. <explanation here>`

Reply:
can u do it?